src/vt-tv/api/info.h

```diff
@@ -538,81 +557,92 @@ struct Info {
     // Vector of tuples of communications to add: {side_to_be_modified, id1, id2, bytes} for an id1 -> id2 communication (1 sends to 2, 2 receives from 1)
     // if type is "sender", communication has to be added to sent communications for object id1
     // if type is "recipient", communication has to be added to received communications for object id2
-    std::vector<std::tuple<std::string, ElementIDType, ElementIDType, double>> communications_to_add;
+    std::vector<std::tuple<std::string, ElementIDType, ElementIDType, double>>
+      communications_to_add;
 
     auto phase_objects = createPhaseObjectsMapping(phase);
     // Checking all communications for object A in all objects of all ranks at given phase: A <- ... and A -> ...
     for (auto& [A_id, object_work] : phase_objects) {
-//      fmt::print("- Object ID: {}\n", A_id);
+      //      fmt::print("- Object ID: {}\n", A_id);
```

Comment:
Weird spacing in the comments here. @pierrepebay Can we delete these?

Reply:
extra spaces removed